docs/topic_guides/comparators.md

```diff
@@ -3,25 +3,166 @@ tags:
+  - Damerau-Levenshtein
+  - Jaro
   - Jaro-Winkler
   - Jaccard
 ---
-# Comparators
+# String Comparators
 
+A Splink model contains a collection of `Comparisons` and `ComparisonLevels` organised in a hierarchy.  For example:
 
-String comparators are algorithms used in data linkage to compare and evaluate the similarity between two strings of text. These algorithms can be applied to different types of data, including names, addresses, and other identifying information, to determine if two records refer to the same individual or entity.
+```
+Data Linking Model
+├─-- Comparison: Date of birth
+│    ├─-- ComparisonLevel: Exact match
+│    ├─-- ComparisonLevel: Up to one character difference
+│    ├─-- ComparisonLevel: Up to three character difference
+│    ├─-- ComparisonLevel: All other
+├─-- Comparison: Name
+│    ├─-- ComparisonLevel: Exact match on first name and surname
+│    ├─-- ComparisonLevel: Exact match on first name
+│    ├─-- etc.
+```
 
-One common use of string comparators in data linkage is to identify duplicates within a dataset. By comparing the strings of two records and calculating their similarity, the algorithm can flag records that may be duplicates and require further investigation. This can help to improve the accuracy and completeness of the data by eliminating incorrect or redundant information.
+For more detail on how comparisons are constructed, see the dedicated [topic guide](customising_comparisons.ipynbcustomising_comparisons.ipynb) as well as fuller descriptions of [`Comparisons`](../comparison.md) and [`Comparison Levels`](../comparison_level.md). 
 
-Another use of string comparators in data linkage is to merge different datasets or databases. By comparing strings across multiple sources, the algorithm can identify records that refer to the same individual or entity and merge them into a single record. This can be useful for creating a more comprehensive and accurate view of the data, and for identifying trends and patterns that may not be visible in a single dataset.
+Within `Comparisons` it is useful for different `Comparison Levels` to allow for different styles (and levels) fuzzy match. Each of these `Comparison Levels` indicates a different class of match between two records and therefore a different type (and amount) of evidence for or against the two records being a match. Once these `Comparison Levels` have been defined, the Splink model is trained to estimate the Match Weight to assign to each `Comparison Level`.
 
+There are a number of string comparator functions available in Splink that allow fuzzy matching for strings within in `Comparisons` and `Comparison Levels`. For each of these fuzzy matching functions, below you will find explanations of how they work, worked examples and recommendations for the types of data they are useful for.
 
-## Jaro similarity
+For guidance on how to choose the most suitable string comparator, and associated threshold, see the dedicated [topic guide]().
```

Comment:
The link to the 'topic guide' is circular and sends you back to the page you're already on. I'm guessing it should be here ... http://127.0.0.1:8000/topic_guides/choosing_comparators.html